app/controllers/chef_profiles_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ChefProfilesController < ApplicationController
+  before_action :set_chef_profile, only: :show
+
+  def index
+    @chef_profiles = ChefProfile.all
+  end
+
+  def show
```

Comment:
should have an instance variable here right?

@chef_profile = ChefProfile.find(params[:id])

Reply:
before_action :set_chef_profile, only: :show

def chef_profile_params
    params.require(:chef_profile).permit(:user_id)
  end

Reply:
I think it can be `before_action :set_chef_profile, only: [:show, :index`].

Because we want users to be able to view the list of chef profiles, as well as individual ones right?

Reply:
We just don't want the users to be able to create new ones or book them without logging in

Reply:
right :)